iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/task/DropPipeProcedureV2.java

```diff
@@ -86,16 +85,11 @@ protected void executeFromWriteConfigNodeConsensus(ConfigNodeProcedureEnv env)
   }
 
   @Override
-  protected void executeFromOperateOnDataNodes(ConfigNodeProcedureEnv env)
-      throws PipeException, IOException {
+  protected void executeFromOperateOnDataNodes(ConfigNodeProcedureEnv env) throws PipeException {
     LOGGER.info("DropPipeProcedureV2: executeFromOperateOnDataNodes({})", pipeName);
 
-    TSStatus result = pushPipeMetaToDataNodes(env);
-    if (result.getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-      throw new PipeException(
-          String.format(
-              "Failed to drop pipe %s on data nodes. Failures: %s", pipeName, result.getMessage()));
-    }
+    // Will not rollback
+    pushPipeMetaToDataNodesIgnoreException(env);
```

Comment:
Why make a change here? The PipeException thrown here is used to trigger retries.